frontend2/src/utils/api.ts

```diff
@@ -0,0 +1,729 @@
+import { ApiApi } from "./types/api/ApiApi";
+import Cookies from "js-cookie";
+import * as $ from "jquery";
+import * as models from "./types/model/models";
+
+// hacky, fall back to localhost for now
+const baseUrl = process.env.REACT_APP_BACKEND_URL || "http://localhost:8000";
+const LEAGUE = 0;
+const PAGE_SIZE = 10;
+
+// Safe vs. unsafe APIs... safe API has been tested, unsafe has NOT
+// TODO: how does url work? @index.tsx?
```

Comment:
has this question been resolved? if not, could you clarify it?

Reply:
I think in old frontend this was set in index.js? So for now should work bc I collect to default staging backend URL with `|| "http://localhost:8000"` but something we need to look at

Reply:
ok got it, we can take a look at this when we work on deployign things to staging/prod